scipy/stats/tests/test_stats.py

```diff
@@ -1626,6 +1626,21 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[1], z1_expected)
 
 
+    def test_mad(self):
+        dat = np.array([2.20, 2.20, 2.4, 2.4, 2.5, 2.7, 2.8, 2.9, 3.03,
+                3.03, 3.10, 3.37, 3.4, 3.4, 3.4, 3.5, 3.6, 3.7, 3.7,
+                3.7, 3.7,3.77, 5.28, 28.95])
+        assert_almost_equal(stats.mad(dat, axis=None), 0.52632)
+
+        dat = dat.reshape(6, 4)
+
+        mad = stats.mad(dat, axis=0)
+        mad_expected = np.asarray([0.644931, 0.7413  , 0.66717 , 0.59304 ])
+        assert_array_almost_equal(mad, mad_expected)
+
+
+
+
 class _numpy_version_warn_context_mgr(object):
     """
     A simple context maneger class to avoid retyping the same code for
```

Comment:
I realize this is not part of your PR but there is a typo in the docstring `maneger` should be `manager` maybe add the fix into your PR?

Reply:
Fixed this in the latest PR.